js/querystring.js

```diff
@@ -0,0 +1,68 @@
+/*!
```

Comment:
I don't think you actually need this anymore, but I moved it to its own module regardless. Inside script.js, I'm using the Node `querystring` module for parsing instead.